plugins/global-search/src/components/DevToolsScene.tsx

```diff
@@ -0,0 +1,187 @@
+import { useMemo, useState } from "react"
+import { cn } from "../utils/className"
+import { type IndexEntry } from "../utils/indexer/types"
+import { useIndexer } from "../utils/indexer/useIndexer"
+
+export function DevToolsScene() {
+    const { index, isIndexing, indexerInstance } = useIndexer()
+
+    const [selectedEntry, setSelectedEntry] = useState<IndexEntry | null>(null)
+
+    const [filterQuery, setFilterQuery] = useState("")
+
+    const entries = useMemo(() => Object.values(index), [index])
+
+    const filteredEntries = useMemo(
+        () =>
+            entries.filter(entry => {
+                if (entry.id === filterQuery) return true
+                switch (entry.type) {
+                    case "CollectionItem":
+                        return Object.values(entry.fields).some(field =>
+                            field.toLowerCase().includes(filterQuery.toLowerCase())
+                        )
+                    default:
+                        return entry.name?.toLowerCase().includes(filterQuery.toLowerCase())
+                }
+            }),
+        [entries, filterQuery]
+    )
+
+    const stats = useMemo(
+        () => ({
+            total: entries.length,
+            byType: entries.reduce(
+                (acc, entry) => {
+                    acc[entry.type] = (acc[entry.type] ?? 0) + 1
+                    return acc
+                },
+                {} as Record<string, number>
+            ),
+        }),
+        [entries]
+    )
+
+    return (
+        <div className="flex flex-col h-full">
```

Comment:
I know its just temporary / for debugging but its a lil completely broken on dark mode :D


<img width="407" height="661" alt="image" src="https://github.com/user-attachments/assets/c72a592b-fe7d-48be-8db5-33e73bd53c92" />

Reply:
I'm not investing time into this for now, as it stays only for a matter of days...